### PR TITLE
[DOC] Draft linux install for Tempo

### LINKED ISSUE #1607: [docs] Add information about the linux binary available under release assets
**hi, thanks for this amazing repo!**
here is the problem:
my application process running on a  2C2G linux machine, which would be deployed privately on customer's machine
is there any way i can install tempo without those huge dependency like docker/k8s
i can't find in [docs](https://grafana.com/docs/tempo/latest/getting-started/example-demo-app/)

### PR DESCRIPTION
**Pull request by knylander-grafana**
**What this PR does**:

Adds Linux Set up instructions for Tempo (based upon the GET procedure). This content also discusses installing from the Linux packages available in the Assets section of the Releases. 

**Which issue(s) this PR fixes**:
Fixes #1607 #836 #1959

**Checklist**
- [ ] Tests updated
- [X] Documentation added
- [ ] `CHANGELOG.md` updated - the order of entries should be `[CHANGE]`, `[FEATURE]`, `[ENHANCEMENT]`, `[BUGFIX]`

### CODE REVIEW

File: docs/tempo/website/setup/_index.md [hunk 1]
```diff
@@ -5,36 +5,34 @@ description: Learn how to set up a Tempo cluster and visualize data
 weight: 150
 ---
 
-# Set up a Tempo cluster
+# Set up Tempo
```
Comment (knylander-grafana) [inline docs/tempo/website/setup/_index.md:8]:
```suggestion
# Set up a Tempo server
```

File: docs/tempo/website/setup/_index.md [hunk 2]
```diff
@@ -1,40 +1,38 @@
...
-menuTitle: Set up a Tempo cluster
-description: Learn how to set up a Tempo cluster and visualize data
+title: Set up a Tempo server or cluster
+menuTitle: Set up a Tempo server or cluster
+description: Learn how to set up a Tempo server or cluster and visualize data
 weight: 150
 ---
 
-# Set up a Tempo cluster
+# Set up a Tempo server or cluster
```
Comment (joe-elliott) [inline docs/tempo/website/setup/_index.md:8]: I see you've added "server" in quite a few spots but it sounds weird to me. We don't like just "cluster"?

File: docs/tempo/website/setup/_index.md [hunk 3]
```diff
@@ -1,40 +1,38 @@
...
 
 Tempo is available as a pre-compiled binary, a Docker image, and as common OS-specific packaging.
 
-No matter which option you choose, setting up your Tempo cluster involves: 
+You can choose to install Tempo on a single system or as a cluster.
```
Comment (joe-elliott) [inline docs/tempo/website/setup/_index.md:12]: You can choose to install Tempo as single service or a distributed set of services?

File: docs/tempo/website/setup/_index.md [hunk 4]
```diff
@@ -1,40 +1,38 @@
...
-This page highlights these steps; more detailed instructions are available on the procedures for deploying clusters.
+This page highlights these steps; more detailed instructions are available on the procedures for installing Tempo.
 
 ## Name your cluster
 
-No matter which deployment method you choose, you need to decide what to call your Tempo cluster.
+If you install Tempo in a cluster, you need to decide what to call it.
 A cluster name must meet the following criteria:
 
 - is 3 to 63 characters long
```
Comment (joe-elliott) [inline docs/tempo/website/setup/_index.md:21]: where do these restrictions come from?

File: docs/tempo/website/setup/_index.md [hunk 5]
```diff
@@ -1,40 +1,38 @@
...
 
-<!-- - [Deploy on Linux]({{< relref "linux">}}) -->
+- [Deploy on Linux]({{< relref "linux">}})
 - [Deploy on Kubernetes using Tanka]({{< relref "tanka">}})
 
-## Test your cluster
+## Test your installation
 
-Once your cluster is deployed, you can test your cluster by visualizing the traces data with a simple TNS app. 
+Once Tempo is deployed, you can test your cluster by visualizing the traces data with a simple TNS app.
```
Comment (joe-elliott) [inline docs/tempo/website/setup/_index.md:35]: we should point people away from TNS. we've moved to this project and it is much simpler to setup and produces better traces:

https://github.com/grafana/tempo/blob/main/example/docker-compose/gcs/docker-compose.yaml#L25

File: docs/tempo/website/setup/_index.md (hunk 2 above)
Comment (knylander-grafana) [reply docs/tempo/website/setup/_index.md:8]: The Tanka instructions set up a cluster while the linux instructions set up a single node. Cluster is no longer accurate for both instructions.

File: docs/tempo/website/setup/_index.md (hunk 5 above)
Comment (knylander-grafana) [reply docs/tempo/website/setup/_index.md:35]: @hedss  FYI.

Comment (knylander-grafana) [reply docs/tempo/website/setup/_index.md:35]: We should create a getting started using that example as a set up. We have an example app set up using the TNS app. I'll create a GH issue for this so we can make that happen.

File: docs/tempo/website/setup/_index.md (hunk 3 above)
Comment (knylander-grafana) [reply docs/tempo/website/setup/_index.md:12]: This brings up a good point. I don't think we have instructions on using Tempo as a distributed set of services. We have the Tanka setup for a cluster and the linux setup for a single node.

File: docs/tempo/website/setup/_index.md (hunk 4 above)
Comment (knylander-grafana) [reply docs/tempo/website/setup/_index.md:21]: They are actually from the [Enterprise Traces content](https://grafana.com/docs/enterprise-traces/latest/setup/). Is this incorrect for a Tempo cluster?

File: docs/tempo/website/setup/_index.md (hunk 3 above)
Comment (knylander-grafana) [reply docs/tempo/website/setup/_index.md:12]:
```suggestion
This section describes how to set up Tempo on a single Linux node or as a cluster using Kubernetes and Tanka. Tempo can also be set up as a distributed set of services.  
```

File: docs/tempo/website/setup/_index.md (hunk 2 above)
Comment (joe-elliott) [reply docs/tempo/website/setup/_index.md:8]: Yeah. I agree. It's a little weird to me that these words "server" and "cluster" basically line up with "single binary" and "distributed" which we use elsewhere in the docs, but see what you're getting at.

FWIW there's nothing that prevents someone from running a Tempo cluster directly on a linux machine (as opposed to using some kind of orchestration), but I understand these instructions are focused on the single binary install.

File: docs/tempo/website/setup/_index.md (hunk 3 above)
Comment (joe-elliott) [reply docs/tempo/website/setup/_index.md:12]: i like the clarification in your suggestion. i think this plays into the concerns above. "server" and "cluster" are just different words then we use throughout the rest of the docs.

File: docs/tempo/website/setup/_index.md (hunk 4 above)
Comment (knylander-grafana) [reply docs/tempo/website/setup/_index.md:21]: I'll remove this section.

**Comment by knylander-grafana** [comment]
@hedss  I've added the udpates we had and created the docker-compose example quick start.

File: docs/tempo/website/getting-started/docker-example.md [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+   local-tempo-1        "/tempo -config.file…"   tempo               running             0.0.0.0:3200->3200/tcp, 0.0.0.0:4317-4318->4317-4318/tcp, 0.0.0.0:9411->9411/tcp, 0.0.0.0:14268->14268/tcp
+   ```
+
+1. As part of the docker compose manifest, Grafana is now reachable on port 3000. Open a browser to localhost:3000 and sign in to Grafana.
+
+1. Once logged in, navigate to the **Explore** page, select the **Tempo** data source and select the **Search** tab. Select **Run query** to list the recent traces stored in Tempo. Select one to view the trace diagram:
+    <p align="center"><img src="../../setup/assets/setup-linux-run-query.png" alt="Query example"></p>
+
+## Enable metrics-generator
+
```
Comment (knylander-grafana) [inline docs/tempo/website/getting-started/docker-example.md:56]: @hedss Is this section correct for the docker-compose example?

File: docs/tempo/website/setup/_index.md (hunk 5 above)
Comment (knylander-grafana) [reply docs/tempo/website/setup/_index.md]: @hedss and I added content for the Docker example you pointed to.

**Review by hedss — APPROVED** [review]
Apart from the other comments already raised, LGTM!

File: docs/tempo/website/getting-started/docker-example.md [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+weight: 600
+---
+
+# Quick start for Tempo
+
+The Tempo repository provides [multiple examples](https://github.com/grafana/tempo/tree/main/example/docker-compose) to help you quickly get started using Tempo and tracing data.
+
+Every example has a `docker-compose.yaml` manifest that includes all of the options needed to explore trace data in Grafana, including resource configuration and trace data generation. The Tempo examples running with docker-compose all include a version of Tempo and a storage configuration like Amazon S3 and GCS.
+
+This quick start guide uses the `local` example. Any data is stored locally in the `tempo-data` folder. To learn more, read the [local storage example REAMDE](https://github.com/grafana/tempo/blob/main/example/docker-compose/local).
```
Comment (joe-elliott) [inline docs/tempo/website/getting-started/docker-example.md:14]:
```suggestion
This quick start guide uses the `local` example. Any data is stored locally in the `tempo-data` folder. To learn more, read the [local storage example README](https://github.com/grafana/tempo/blob/main/example/docker-compose/local).
```

**Review by joe-elliott — APPROVED** [review]

File: docs/tempo/website/setup/linux.md [hunk 8]
```diff
@@ -46,243 +43,236 @@ This is not recommended for production deployments. This guide focuses on setup
...
+      local:
+         path: /tmp/tempo/blocks
+      pool:
+         max_workers: 100                 # worker pool determines the number of parallel requests to the object store backend
+         queue_depth: 10000
+
+   overrides:
+   metrics_generator_processors: [service-graphs, span-metrics]
 ```
+>**Note:** that in the above configuration we enable the metrics generator to generate Prometheus metrics data from incoming trace spans. This is sent to a Prometheus remote write compatible metrics store at `http://prometheus:9090/api/v1/write` (in the `metrics_generator` configuration block). Ensure you change the relevant `url` parameter to your own Prometheus compatible storage instance, or disable the metrics generator by replacing `metrics_generator_enabled: true` with `metrics_generator_enabled: false` if you do not wish to generate span metrics.
```
Comment (knylander-grafana) [inline docs/tempo/website/setup/linux.md:142]:
```suggestion
>**Note:** In the above configuration, metrics generator is enabled to generate Prometheus metrics data from incoming trace spans. This is sent to a Prometheus remote write compatible metrics store at `http://prometheus:9090/api/v1/write` (in the `metrics_generator` configuration block). Ensure you change the relevant `url` parameter to your own Prometheus compatible storage instance, or disable the metrics generator by replacing `metrics_generator_enabled: true` with `metrics_generator_enabled: false` if you do not wish to generate span metrics.
```

**Pull request merged**